datachannel.md

```diff
@@ -0,0 +1,171 @@
+   start the presentation with ID and URL for the initial presentation connection.
+2. Receiver spawn a Data Channel on the same RTC connection of control channel, with
+   presentation ID as label.
+3. Get a reply on control channel with the ID and URL to confirm success
+   (or report an error).
+
+## Presentation Resumption
+
+1. Send a control message on control channel from the controller to the display
+   reconnect to a presentation with ID and URL for the initial presentation connection.
+2. Receiver spawn a Data Channel on the same RTC connection of control channel, with
+   presentation ID as label, if there is a corresponding presentation to be resumed.
+2. Get a reply on control channel with the ID, URL, and stream ID to confirm success
+   (or report an error).
+
+## Presentation Connections
+
+Multiple connections from a controller can be handled by creating unique Data
+Channels for each peer. Unique stream ID will be selected for muxing/demuxing
+messages on the underlying SCTP transport, see
+[draft-ietf-rtcweb-data-protocol-09](https://tools.ietf.org/html/draft-ietf-rtcweb-data-protocol-09#section-6).
+
+## Presentation Connection Messaging
+
+Binary or text messages can be sent on the Data Channel corresponding to the
+presentation connection.  The framing of these messages is defined in
+[draft-ietf-rtcweb-data-channel](https://tools.ietf.org/html/draft-ietf-rtcweb-data-channel-13#section-6.6).
+
+Either side may close the Presentation Connection by following procedure:
+1. Send a control message with close reason on control channel to remote peer.
+2. While receiving the control message, initiate the
+   [Data Channel close procedure](https://tools.ietf.org/html/draft-ietf-rtcweb-data-channel-13#section-6.6)
+3. Consider Presentation Connection is closed while Data Channel is closed.
+
+## Presentation Termination
+
+For controller initiated termination:
+1. A controller send a control message with URL and ID on control channel to
+   instruct a display to terminate a presentation.
+2. The receiver sends a reply to all associated control channels to inform all
+   remote peers that a presentation is about to be terminated.
+
+For receiver initiated termination:
+1. The receiver directly sends out reply to all associated control channels to
+   inform all remote peers that a presentation is about to be terminated.
+
+# Remote Playback API functionality
+
+WebRTC framework supports media codec negociation and real-time media transport.
+
+**TODO:** Fill in when Remote Playback equirements are known.
+See [Issue #3](https://github.com/webscreens/openscreenprotocol/issues/3).
+
+# Reliability
+
+WebRTC is designed to be operated on heterogeneous network environment, using
+ICE for address discovery and built-in negociation/re-negociation mechanism.
+
+*TODO:* Get reliability data.
+
+# Latency to establish a new connection
+
+Data on connection establishment latency.
```

Comment:
It sounds like are three sources of latency:
1. Connecting a reliable channel the first time for for ICE negotiation.
2. Establishing the first Data Channel (perhaps this is part of #1).
3. Adding an additional Data Channel on top of an existing peer-to-peer connection.